app/views/external_users/claims/_summary_fees.html.haml

```diff
@@ -36,18 +36,53 @@
       %p
         = t("shared.summary.no_values.#{section}")
   - else
+    - if section.to_s.eql?('basic_fees')
+      %h3.heading-medium
+        = t('shared.summary.additional_fees')
+
     %table.summary
-      %caption
-        %span.table-caption.visuallyhidden
-          = t('.caption')
+      %caption.table-caption.visuallyhidden
+        = t('.caption')
+
+      %thead
+        %tr
+          %th{ scope: 'col' }
+            = t('shared.summary.fee_type')
+          %th{ scope: 'col' }
+            = t('shared.summary.dates')
+          %th.numeric{ scope: 'col' }
+            = t('shared.summary.quantity')
+          %th.numeric{ scope: 'col' }
+            = t('shared.summary.rate')
+          %th.numeric{ scope: 'col' }
+            = t('shared.summary.amount')
+          %th.numeric{ scope: 'col' }
+            = t('shared.summary.vat_amount')
+          %th.numeric{ scope: 'col' }
+            = t('shared.summary.total_amount')
+
       %tbody
         -# The collection of fees
         - present_collection(collection.sort_by(&:position)).each.with_index(1) do |fee, index|
-          %tr
-            %th{ scope: 'row' }
-              = t('shared.summary.fee_index', index: index)
-            %td
-              %table.nested-summary
-                %tbody
-                  -# Each instance of a fee
-                  = render partial: 'external_users/claims/summary_fee', locals: { fee: fee }
+          -# Each instance of a fee
+          = render partial: 'external_users/claims/summary_fee', locals: { fee: fee }
+
+      %tfoot
+        %tr
+          %th{ scope: 'row' }
+            = t('shared.summary.total')
+          %td.numeric
+            = ''
+          %td.numeric
+            = ''
+          %td.numeric
+            = ''
+          %td.numeric
+            = claim.fixed_fees_total if section.to_s.eql?('fixed_fees')
+            = claim.misc_fees_total if section.to_s.eql?('misc_fees')
+          %td.numeric
```

Comment:
Could these be in a loop rather than declared literally? 

Also I think you could use a ternary operator given there are only two possibilities:

```ruby
= section.to_s.eql?('fixed_fees') ? claim.fix_fees_total : claim.misc_fees_total
```